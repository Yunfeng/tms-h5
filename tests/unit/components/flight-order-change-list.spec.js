import { shallowMount } from '@vue/test-utils'
import Component from '@/components/flight-order-change-list.vue'

describe('flight-order-refund-list.vue', () => {
  // test方法是it的别名之一
  test('matches snapshot', () => {
    const wrapper = shallowMount(Component)
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('is a Vue instance', () => {
    const wrapper = shallowMount(Component)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  // it('slot content test', () => {
  //   const defaultSlotContent = 'content for default slot'
  //   const wrapper = shallowMount(Component, {
  //     slots: {
  //       default: defaultSlotContent,
  //       excelFormatDesc: 'excel format desc from excelFormatDesc slot'
  //     }
  //   })

  //   // console.log(wrapper.html())
  //   expect(wrapper.find('#myModalLabel').text()).toEqual(defaultSlotContent);
  // })

})
