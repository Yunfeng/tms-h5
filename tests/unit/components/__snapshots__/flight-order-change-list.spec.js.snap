// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`flight-order-refund-list.vue matches snapshot 1`] = `
<table id="flight-refund-list" class="table table-sm table-striped table-hover">
  <thead>
    <tr>
      <th>改签单号</th>
      <th>票号</th>
      <th></th>
      <th>乘客/公司</th>
      <th>原编码</th>
      <th colspan="6" class="bg-info text-center text-white">行程</th>
      <th class="text-right">改签费</th>
      <th class="text-right">服务费</th>
      <th>新票号</th>
      <th>提交时间</th>
      <th>状态</th>
      <th></th>
    </tr>
  </thead>
  <tbody></tbody>
</table>
`;
